templates/emails/fair.txt

```diff
@@ -0,0 +1,20 @@
+{% extends 'emails/base.txt' %}
```

Comment:
Would be great down the line to have a markdown to txt and to html engine. Not sure how hard that would be to set up with the email template but would be good to be more DRY

Reply:
Yeah, a simpler alternative solution would just be to strip the HTML tags from the HTML response and use that for the text response.

However, there are some slight differences, for example links in HTML can actually be links, but in the text version we tell the user to go to some URL.

I agree, somewhere down the line we could avoid a lot of duplication by combining the HTML and non-HTML templates.